ui/src/overview.tsx

```diff
@@ -291,3 +309,204 @@ function ParticipantCriteria(props: {
+      underlayName: underlayName,
+      entityName: "person",
+      searchEntityCountsRequest: searchEntityCountsRequest,
+    });
+
+    if (!data.counts) {
+      throw new Error(
+        "The counts property returned by the searchEntityCounts API is undefined."
+      );
+    }
+
+    const demographicData = data.counts;
+    let totalCount = 0;
+    const demographicsByGender = new Map();
+    const demographicsByGenderAgeRace = new Map();
+    const demographicRaces = new Set();
+
+    for (let i = 0; i < demographicData.length; i++) {
+      const count = demographicData[i].count ?? 0;
+      const gender = demographicData[i].definition?.gender.stringVal ?? "Other";
+      const race = demographicData[i].definition?.race.stringVal ?? "Other";
+      const currentYear = new Date().getFullYear();
+      const yob = demographicData[i].definition?.year_of_birth?.int64Val ?? 0;
+      const age = parseInt(currentYear.toString()) - yob;
+      let ageRange = "";
+      if (age >= 65) {
+        ageRange = "65+";
+      } else if (age >= 45 && age <= 64) {
+        ageRange = "45-64";
+      } else if (age <= 44) {
+        ageRange = "18-44";
+      }
+
+      // Accumulate total count for results
+      totalCount += count;
+
+      // Configure hash map to represent data for gender chart
+      if (!demographicsByGender.has(gender)) {
+        demographicsByGender.set(gender, 0);
+      }
+      demographicsByGender.set(
+        gender,
+        demographicsByGender.get(gender) + count
+      );
+
+      // Configure hash map to represent data for gender / age / race chart
+      if (!demographicsByGenderAgeRace.has(`${gender} ${ageRange}`)) {
+        demographicsByGenderAgeRace.set(`${gender} ${ageRange}`, new Map());
+      }
+
+      if (!demographicsByGenderAgeRace.get(`${gender} ${ageRange}`).has(race)) {
+        demographicsByGenderAgeRace.get(`${gender} ${ageRange}`).set(race, 0);
+      }
+
+      const currCountForRace = demographicsByGenderAgeRace
+        .get(`${gender} ${ageRange}`)
+        .get(race);
+
+      demographicsByGenderAgeRace
+        .get(`${gender} ${ageRange}`)
+        .set(race, currCountForRace + count);
+
+      // Configure set of races returned in results
+      if (!demographicRaces.has(race)) {
+        demographicRaces.add(race);
+      }
+    }
+
+    return {
+      totalCount,
+      demographicsByGender: Array.from(demographicsByGender, ([key, value]) => {
+        return {
+          name: key,
+          count: value,
+          color: barColours[0],
+        };
+      }),
+      demographicRaces: Array.from(demographicRaces),
+      demographicsByGenderAgeRace: Array.from(
+        demographicsByGenderAgeRace,
+        ([key, value]) => {
+          return {
+            ageRange: key,
+            ...Object.fromEntries(value),
+          };
+        }
+      ),
+    };
+  }, [cohort]);
+
+  const demographicData = useAsyncWithApi(fetchDemographicData);
```

Comment:
`demographicState` or `fetchState` or similar is a better name for this since it encompasses more things, including `demographicData.data` which is actually the data.